src/transactionsFactories/transferTransactionsFactory.ts

```diff
@@ -89,27 +155,184 @@ export class NextTransferTransactionsFactory {
         }).build();
     }
 
+    createEGLDTransfer(args: {
```

Comment:
Add comments that they are legacy functions? Since old constructor is not deprecated, it's OK not to deprecate these, as well, but at least mark them with "This is a legacy function, use ... instead".

Reply:
Added